tiledb/sm/serialization/array_schema.cc

```diff
@@ -1830,11 +1817,10 @@ deserialize_load_array_schema_response(
   try {
     switch (serialization_type) {
       case SerializationType::JSON: {
-        ::capnp::JsonCodec json;
         ::capnp::MallocMessageBuilder message_builder;
         auto builder =
             message_builder.initRoot<capnp::LoadArraySchemaResponse>();
-        json.decode(kj::StringPtr(data.data()), builder);
```

Comment:
You can validate that I did not miss a case by seeing there are zero occurences of `kj::StringPtr` in the codebase.